sqlglot/dialects/db2.py

```diff
@@ -0,0 +1,227 @@
+            "ORDER BY": TokenType.ORDER_BY,
+        }
+
+    class Parser(parser.Parser):
+        ALTER_TABLE_ADD_REQUIRED_FOR_EACH_COLUMN = False
+        WINDOW_BEFORE_PAREN_TOKENS = {TokenType.OVER, TokenType.KEEP}
+        VALUES_FOLLOWED_BY_PAREN = False
+        ALIAS_POST_TABLESAMPLE = True
+        STRING_ALIASES = True
+
+        FUNCTIONS = {
+            **parser.Parser.FUNCTIONS,
+            "DATE_SUB": build_date_delta_with_interval(exp.DateSub),
+            "TO_CHAR": _build_timetostr_or_tochar,
+            "DATE_ADD": build_date_delta_with_interval(exp.DateAdd),
+            "DATE": lambda args: exp.TsOrDsToDate(this=seq_get(args, 0)),
+            "CHAR": lambda self: self._parse_chr(),
+            "GROUP_CONCAT": lambda self: self._parse_group_concat(),
+            # https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_values
+            "VALUES": lambda self: self.expression(
+                exp.Anonymous, this="VALUES", expressions=[self._parse_id_var()]
+            ),
+            "TO_DATE": build_formatted_time(exp.StrToDate, "db2"),
+            "DAY": lambda args: exp.Day(this=exp.TsOrDsToDate(this=seq_get(args, 0))),
+            "TO_DAYS": lambda args: exp.paren(
+                exp.DateDiff(
+                    this=exp.TsOrDsToDate(this=seq_get(args, 0)),
+                    expression=exp.TsOrDsToDate(this=exp.Literal.string("0000-01-01")),
+                    unit=exp.var("DAY"),
+                )
+                + 1
+            ),
+        }
+
+        NO_PAREN_FUNCTIONS = {
+            **parser.Parser.NO_PAREN_FUNCTIONS,
+            TokenType.CURRENT_DATE: exp.CurrentDate,
+            TokenType.CURRENT_TIME: exp.CurrentTime,
+            TokenType.CURRENT_TIMESTAMP: exp.CurrentTimestamp,
+        }
+        FUNCTION_PARSERS: t.Dict[str, t.Callable] = {
+            **parser.Parser.FUNCTION_PARSERS,
+            "JSON_ARRAY": lambda self: self._parse_json_array(
+                exp.JSONArray,
+                expressions=self._parse_csv(lambda: self._parse_format_json(self._parse_bitwise())),
+            ),
+            "JSON_ARRAYAGG": lambda self: self._parse_json_array(
+                exp.JSONArrayAgg,
+                this=self._parse_format_json(self._parse_bitwise()),
+                order=self._parse_order(),
+            ),
+            "JSON_OBJECT": lambda self: self.elf._parse_json_object(
+                exp.JSONObject,
+                this=self._parse_format_json(self._parse_bitwise()),
+            ),
+            "XMLTABLE": lambda self: self._parse_xml_table(),
+        }
+        DISTINCT_TOKENS = {TokenType.DISTINCT}
+
+        QUERY_MODIFIER_PARSERS = {
+            **parser.Parser.QUERY_MODIFIER_PARSERS,
+            TokenType.ORDER_BY: lambda self: ("order", self._parse_order()),
+        }
+
+    class Generator(generator.Generator):
+        # Specifies how AST nodes, i.e. subclasses of exp.Expression, should be converted into SQL
```

Comment:
```suggestion
```

Reply:
Comment removed in 1e96557b9b8456adb61c6cc1eb879902bf2ae9d6